src/render_cmd.rs

```diff
@@ -168,67 +188,98 @@ impl RenderCmd {
 
     /// Execute the commands in the given plan
     fn exec_plan(&self, plan: &Plan) -> anyhow::Result<()> {
-        for (deployment, cmd) in &plan.commands {
-            info!(" - {}", deployment);
+        if !&plan.pre_commands.is_empty() {
```

Comment:
Is the check for empty `pre_commands` better off at the `plan` stage where you are checking if the `update_dependencies` flag is there? ie after here https://github.com/hendrikmaus/helm-templexer/pull/60/files#diff-a374dd3d48a9861daacaafcecde4f50fcf330ea0e490f16cf6359f564f2d5c9fR103

Reply:
The plan phase populates the `pre_commands` map, and here we check if there is anything to execute.